### ISSUE TITLE
Unable to get a model after pruning, only a mask and a model with sparse weight.

### ISSUE THREAD

**Issue by qi657** [microsoft/nni#4686]
**Describe the issue**:
[2022-03-23 16:07:48] INFO (nni.compression.pytorch.speedup.compressor/MainThread) resolve the mask conflict
[2022-03-23 16:07:48] INFO (nni.compression.pytorch.speedup.compressor/MainThread) replace compressed modules...
[2022-03-23 16:07:48] INFO (nni.compression.pytorch.speedup.compressor/MainThread) Warning: cannot replace (name: .aten::slice.22, op_type: aten::slice) which is func type
[2022-03-23 16:07:48] INFO (nni.compression.pytorch.speedup.compressor/MainThread) Warning: cannot replace (name: .aten::slice.26, op_type: aten::slice) which is func type
[2022-03-23 16:07:48] INFO (nni.compression.pytorch.speedup.compressor/MainThread) Warning: cannot replace (name: .aten::slice.23, op_type: aten::slice) which is func type
[2022-03-23 16:07:48] INFO (nni.compression.pytorch.speedup.compressor/MainThread) Warning: cannot replace (name: .aten::slice.27, op_type: aten::slice) which is func type
[2022-03-23 16:07:48] INFO (nni.compression.pytorch.speedup.compressor/MainThread) Warning: cannot replace (name: .aten::slice.24, op_type: aten::slice) which is func type
[2022-03-23 16:07:48] INFO (nni.compression.pytorch.speedup.compressor/MainThread) Warning: cannot replace (name: .aten::slice.28, op_type: aten::slice) which is func type
[2022-03-23 16:07:48] INFO (nni.compression.pytorch.speedup.compressor/MainThread) Warning: cannot replace (name: .aten::slice.25, op_type: aten::slice) which is func type
[2022-03-23 16:07:48] INFO (nni.compression.pytorch.speedup.compressor/MainThread) Warning: cannot replace (name: .aten::slice.29, op_type: aten::slice) which is func type
[2022-03-23 16:07:48] INFO (nni.compression.pytorch.speedup.compressor/MainThread) speedup done
**Unable to get a model after pruning, only a mask and a model with sparse weight?**

**Environment**:
- NNI version:2.5
- Training service (local|remote|pai|aml|etc):local
- Client OS:
- Server OS (for remote mode only):
- Python version:3.6
- PyTorch/TensorFlow version: torch1.6
- Is conda/virtualenv/venv used?: yes
- Is running in Docker?: No

**Configuration**:
 - Experiment config (remember to remove secrets!):
 - Search space:

**Log message**:
 - nnimanager.log:
 - dispatcher.log:
 - nnictl stdout and stderr:
 

**How to reproduce it?**:

**Comment by scarlett2018** [comment]
@qi657 - you will need to do one more step to speedup the model with the masks using NNI's modelspeedup tool, refer to: https://nni.readthedocs.io/en/stable/Compression/ModelSpeedup.html

**Comment by scarlett2018** [comment]
@qi657 - I'm closing the issue as the question has been answered. feel free to reopen it if the answer does not work for you.

**Comment by qi657** [comment]
> @qi657 - I'm closing the issue as the question has been answered. feel free to reopen it if the answer does not work for you.

ok，thanks！
